conans/test/integration/options_test.py

```diff
@@ -7,6 +7,52 @@
 
 class OptionsTest(unittest.TestCase):
 
+    def general_scope_options_test_package_test(self):
+        client = TestClient()
+        conanfile = """from conans import ConanFile
+class Pkg(ConanFile):
+    options = {"shared": ["1", "2"]}
+    def configure(self):
+        self.output.info("BUILD SHARED: %s" % self.options.shared)
+"""
+        test = """from conans import ConanFile
+class Pkg(ConanFile):
+    def test(self):
+        pass
+"""
+        client.save({"conanfile.py": conanfile})
+        client.run("create . Pkg/0.1@user/testing -o *:shared=1")
+        self.assertIn("Pkg/0.1@user/testing: BUILD SHARED: 1", client.out)
+        client.run("create . Pkg/0.1@user/testing -o shared=2")
```

Comment:
I think unscoped options' bheaviour is a bit weird for users. We should remove this from the docs (if any) and mark this to be deprecated in conan 2.0 or whenever